### PR TITLE
Made the legacy map import more robust

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/10096.

### CODE REVIEW

File: OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs [hunk 1]
```diff
@@ -60,7 +60,10 @@ public virtual void Run(ModData modData, string[] args)
 				var format = GetMapFormatVersion(basic);
 				ValidateMapFormat(format);
 
-				var tileset = GetTileset(mapSection);
+				var tileset = GetTileset(mapSection).ToUpperInvariant(); // the original game isn't case sensitive, but we are
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs]: Comments on a new line and starting with a capital letter, please.

**Comment by abcdefg30** [comment]
Looks good to me otherwise. :+1: / :white_check_mark:

File: OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs]: Okay.

**Comment by GraionDilach** [comment]
:+1:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/397a476acc8f07e91e332786442218e533313d59...ea7b0c9f7963a419ed72e4f98cecd85820c5108f)

**Pull request merged**